### ISSUE TITLE
How to change the text of the FooterCount (i18n)?

### ISSUE THREAD

**Issue by djalmajr** [jossmac/react-images#354]
**Steps to reproduce the behavior:**

My snippet of code is this:

![image](https://user-images.githubusercontent.com/1858147/76144175-974ec100-605c-11ea-8237-64d76a5a1911.png)

**Actual behavior:**

I would like to change only this part of the text

![image](https://user-images.githubusercontent.com/1858147/76144153-5a82ca00-605c-11ea-919c-56be37494359.png)

**Expected behavior:**

To this: 

![image](https://user-images.githubusercontent.com/1858147/76144167-81d99700-605c-11ea-89e2-ccd229d1d2e1.png)

How can I achieve this? I tried using formatters but no luck :/

Would be nice if we have i18n... By the way, thanks for this awesome lib

**Comment by martinjuhasz** [comment]
I had the same requirement. There is no way to just translate the `of` part of the footer. 
You need to add a custom Component and replace it with the default `FooterCount` one.

What i did:

```
const CustomModalFooter = ({ currentIndex, views }) => {
	const activeView = currentIndex + 1
	const totalViews = views.length

	if (!activeView || !totalViews) return null
	return (
		<span>
			{activeView} von {totalViews}
		</span>
	)
}
```
```
<Carousel
    components={{ FooterCount: CustomModalFooter }}
    views={[...]}
/>
```

**Comment by anhtran** [comment]
I agree with @martinjuhasz, you should re-write the component to get that.

**Comment by djalmajr** [comment]
So, where this is used:

![image](https://user-images.githubusercontent.com/1858147/76316092-bb0e4300-62b8-11ea-95d8-891e84869797.png)

I thought it would be used for this kind of problem. Re-writting the code breaks the DRY for a simple thing. :/

Anyway, thanks for response :)

**Comment by martinjuhasz** [comment]
These props are not used for the `FooterCount` component. The `FooterCount` component does not use the `formatters` feature [as you can see here](https://github.com/jossmac/react-images/blob/master/src/components/Footer.js#L116), although adding support for it would be fairly easy.

The `formatters` you highlighted above are used for alt-texts on the buttons for next and previous images. When you hover your mouse on those buttons the text will appear.

I gave the formatters props a try and somehow it only works when declaring ALL formatters, not just a single one. From a quick look at the code it should be merged, but it only worked for me when delcaring all formatter like:

```
<Carousel
    formatters={{
        getAltText: () => 'a',
        getNextLabel: () => 'b',
        getPrevLabel: () => 'c',
        getNextTitle: () => 'd',
        getPrevTitle: () => 'e',
        getCloseLabel: () => 'f',
        getFullscreenLabel: () => 'g',
    }}
    ...
/>
```
